CHANGES.txt

```diff
@@ -70,6 +70,9 @@ Breaking Changes
 Changes
 =======
 
+- Fully qualified column names including a schema name will no longer match on
```

Comment:
should this be a breaking change?

Reply:
Hm not sure. It's not valid semantics so it could also be regarded as a fix. I choose the middle but can also change it.

Reply:
Fair enough, I see where you're coming from. 
I mentioned it as, even though it would be a weird thing, maybe some customers are relying on the previous behaviour?

Reply:
If we're worried about this breaking behavior, I'd suggest not backporting it to `2.3`.

I see it as a bug though since it violates SQL compliance.